core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -1339,6 +1342,20 @@ public static boolean useLegacyRouteDecoratorOrdering() {
         return DEFAULT_USE_LEGACY_ROUTE_DECORATOR_ORDERING;
     }
 
+    /**
+     * Returns whether to allow double dots ({@code ..}) in query string.
```

Comment:
in a request path query string (because we allow double dots in a form data anyway)